physics/discrete_trajectory.hpp

```diff
@@ -131,6 +131,14 @@ class DiscreteTrajectory : public Forkable<DiscreteTrajectory<Frame>,
   // |time|.  This trajectory must be a root.
   void ForgetBefore(Instant const& time);
 
+  // This trajectory must be root, and must not be already downsampling.
+  // Following this call, this trajectory must not have forks when calling
+  // |Append|.  Occasionally removes intermediate points from the trajectory
+  // when |Append|ing, ensuring that |EvaluatePosition| returns a result within
+  // |tolerance| of the missing points.  |max_dense_intervals| is the largest
+  // number of points that can be added before removal is considered.
```

Comment:
Is it the number of *points* or the number of *intervals*?

Reply:
After the first round of downsampling, this is the same, because "added" (we only add right endpoints).
I'm not sure how to word that properly, we add points, not intervals.